### DISCUSSION TITLE
error TS6200: Definitions of the following identifiers conflict with those in another file

### DISCUSSION THREAD

**Discussion by nclemeur** [vuejs/language-tools#4230] [Q&A]
I am upgrading my project to vue-tsc 2.0.11 (from 1.8.27) and I am facing some issue with emitting the types when using monorepos (yarn workspaces). I have several libs that depend on each other and for each of these I am using `vue-tsc --declaration --emitDeclarationOnly` to generate the *.d.ts files. Now it seems that for the first *.vue file in each project,  some sort of global definition is generated. It looks like this
```
export declare const __VLS_globalTypesStart: {};
declare global {
...
export declare const __VLS_globalTypesEnd: {};
```

Unoftunatelly this seems to be generated in each sub-project, and I am having the following error on dependent project:
```
error TS6200: Definitions of the following identifiers conflict with those in another file: __VLS_IntrinsicElements, __VLS_Element, __VLS_GlobalComponents, __VLS_IsAny, __VLS_PickNotAny, __VLS_intrinsicElements, __VLS_SelfComponent, __VLS_WithComponent, __VLS_FillingEventArg_ParametersLength, __VLS_FillingEventArg, __VLS_FunctionalComponentProps, __VLS_AsFunctionOrAny, __VLS_UnionToIntersection, __VLS_OverloadUnionInner, __VLS_OverloadUnion, __VLS_ConstructorOverloads, __VLS_NormalizeEmits, __VLS_PrettifyGlobal
```

I did not see these issue with vue-tsc 1.8.x

Any idea on how to resolve this problem?

**Comment by nclemeur-daesim** [comment]
Looks like this has been marked as a bug (see #4231)
